vendor/bat-native-ads/include/bat/ads/build_channel_constants.h

```diff
@@ -0,0 +1,19 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_VENDOR_BAT_NATIVE_ADS_INCLUDE_BAT_ADS_BUILD_CHANNEL_CONSTANTS_H_
```

Comment:
hmm, we do not have any constants here. Is `_constants.h` file name relevant here?

Reply:
see DM. thanks